bucket/iphone-whatsapp-to-android-transfer.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "3.2.130",
+    "description": "Transfer Whatsapp Chat History from iPhone to Android",
+    "homepage": "https://www.backuptrans.com/iphone-whatsapp-to-android-transfer.html",
+    "license": "Unknown",
+    "architecture": {
+        "64bit": {
+            "url": "https://www.backuptrans.com/download/exe/iphone-whatsapp-to-android-transfer_x64.exe#/dl.7z",
+            "hash": "5fba3d6ad834fb4a9a30d809d22def27387503768920eea31e3499c6c1aa8e29"
+        },
+        "32bit": {
+            "url": "https://www.backuptrans.com/download/exe/iphone-whatsapp-to-android-transfer.exe#/dl.7z",
+            "hash": "a25535021b3efd4da15d28b5d27c9853472e3ef1add7d7d6a38eaeb039b973ac"
+        }
+    },
+    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst*nsis\" -Recurse",
+    "notes": "Without iTunes application (https://support.apple.com/downloads/itunes) installed there is not possible to detect iPhone devices.",
+    "shortcuts": [
+        [
+            "iPhoneWhatsappToAndroidTransfer.exe",
+            "Backuptrans iPhone Whatsapp to Android Transfer"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.backuptrans.com/download.html",
+        "regex": "(?sm)Backuptrans iPhone Whatsapp to Android Transfer.*?V([\\d.]+)"
```

Test you autoupdates before posting PR

```suggestion
        "regex": "(?sm)Backuptrans iPhone WhatsApp to Android Transfer.*?V([\\d.]+)"
```